woocommerce/woocommerce
# In-App Marketplace: Update "no results" design for Extensions search

### Changes proposed in this Pull Request:

Updates the design of the WooCommerce > Extensions search "no results" page. Closes [WCCOM#18191](https://github.com/Automattic/woocommerce.com/issues/18191). Builds atop #39690.

Sample screenshot (note that the category selector shown in the screenshot now won't appear; see comments for discussion):

![Screenshot 2023-09-19 at 09-32-23 Extensions ‹ WooCommerce ‹ woocommerce http __localhost 8888_](https://github.com/woocommerce/woocommerce/assets/53293/f7c68c89-f74d-4bfc-9316-e01d863901e6)



### How to test the changes in this Pull Request:



1. Check out this branch and recompile the assets with e.g. `cd plugins/woocommerce-admin && pnpm start`
2. In your test page, visit the WooCommerce > Extensions page and search for something that doesn't exist. See the new design.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Update the design of the extensions search "no results" page.

</details>
